reference/v8.json

```diff
@@ -790,15 +796,24 @@
       "property-function": true,
       "values": [
         "map",
-        "viewport"
+        "viewport",
+        "auto"
       ],
-      "doc": "Aligns text to the plane of the `viewport` or the `map` when the map is pitched. Matches `text-rotation-alignment` if unspecified.",
+      "default": "auto",
+      "doc": "Aligns text to the plane of the `viewport` or the `map` when the map is pitched. Matches `text-rotation-alignment` if `auto`.",
```

Comment:
Yeah I see what you mean. In 5ab221e I updated so it now reads:

#### spec docs
<img width="632" alt="screen shot 2016-09-15 at 9 14 40 pm" src="https://cloud.githubusercontent.com/assets/63186/18575212/e1e2b390-7b89-11e6-8bb5-aa6e5d9f04c6.png">

#### iOS SDK docs
<img width="1254" alt="screen shot 2016-09-15 at 9 14 27 pm" src="https://cloud.githubusercontent.com/assets/63186/18575213/e1e46bd6-7b89-11e6-901d-01ff69f59a19.png">